packages/block-library/src/footnotes/index.php

```diff
@@ -105,8 +105,10 @@ function register_block_core_footnotes_post_meta() {
 		}
 	}
 }
-// Most post types are registered at priority 10, so use priority 20 here in
-// order to catch them.
+/**
```

Comment:
@jorgefilipecosta Sorry, I missed this at the time -- comments should start with `/*` (a single asterisk) rather than a double asterisk). The latter is used for docblocks only to allow them to be picked up by parser for developer.wordpress.org 

Are you able to do a follow up commit.

Reply:
Sure doing a follow up fix, thank you for catching this 👍